// [python quickstart](https://grpc.io/docs/quickstart/python.html#run-a-grpc-application)
// python -m grpc_tools.protoc --python_out=. --grpc_python_out=. -I. xxxxxxx.proto


syntax = "proto3";

service InstanceSegm {
    rpc infer_one_img(ISOneImgRequest) returns (ISOneImgReply) {}
}

message ISOneImgRequest {
    bytes img_bgr_bytes = 1;
}

message ISOneImgReply {
    bytes result_bytes = 1;
}


